src/flynt/transform/percent_transformer.py

```diff
@@ -113,11 +114,6 @@ def make_fv(key: str):
             # no match means it's just a literal string
             segments.append(ast.Str(s=block.replace("%%", "%")))
 
-    if mapping:
```

Comment:
Removing this check completely is changing the logic outside of aggressive flag; I maybe will re-add this together with a demonstrating unit test.